platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationImpl.java

```diff
@@ -65,7 +65,7 @@ public ApplicationImpl(Repository repo) {
       repoName = repo.getName();
       features.addAll(Arrays.asList(repo.getFeatures()));
     } catch (Exception e) {
-      throw new RuntimeException(e.getMessage());
+      throw new IllegalStateException(e.getMessage());
```

Comment:
❓ Did we really want to loose the cause by not passing that caught exception to the ctor?

Reply:
I'm a little concerned that we're exposing internals that needn't be exposed here. Better to log the cause and throw the exception sans cause, I think.